examples/with-graphql-hooks/components/post-upvoter.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { useMutation } from 'graphql-hooks'
+
+const UPDATE_POST = `
+  mutation updatePost($id: ID!, $votes: Int) {
+    updatePost(id: $id, votes: $votes) {
+      id
+      __typename
+      votes
+    }
+  }
+`
+
+export default function PostUpvoter ({ votes, id, onUpdate }) {
+  const [updatePost] = useMutation(UPDATE_POST)
+
+  return (
+    <button
+      onClick={async () => {
```

Comment:
Just curious, how dangerous is this as it doesn't handle the `updatePost` / `onUpdate` breaking 🤔

Reply:
You're right. I've wrapped this in a try/catch to provide some very basic error handling.

Reply:
💯